frontend/web/components/segments/CohortSegmentDetail/CohortSegmentDetail.scss

```
@@ -0,0 +1,47 @@
+  }
+
+  &__status {
+    font-size: 11px;
+    padding: 3px 10px;
+    border-radius: 10px;
+  }
+
+  &__status-dot {
+    width: 6px;
+    height: 6px;
+    background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)